web/themes/docsy/layouts/about/list.html

```diff
@@ -5,8 +5,8 @@ <h1>{{ .Title }}</h1>
 	{{ if (and (not .Params.hide_readingtime) (.Site.Params.ui.readingtime.enable)) }}
             {{ partial "reading-time.html" . }}
```

Comment:
Not sure if this is possible but can be keep the TOC for Developer Guide below the short text? If not, no problem.